tests/components/onewire/test_coupler.py

```diff
@@ -0,0 +1,121 @@
+"""Tests for 1-Wire devices connected on OWServer."""
```

Comment:
Please move these tests to the onewire sensor platform tests.

Reply:
@MartinHjelmare tests moved in 45aba1a